tests/cluster/check_remoteaccount.py

```diff
@@ -56,6 +55,39 @@ def stop(self):
             raise Exception("SimpleServer failed to stop quickly")
 
 
+
+class CheckIterWrapper(object):
+    def setup(self):
+        self.line_num = 6
+        self.account = MockAccount()
+        self.account.ssh_capture("mkdir -p /tmp").has_next()
+        self.temp_file = "/tmp/ducktape-test-" + str(random.randint(0, 100000))
+        for i in range(self.line_num):
+            self.account.ssh_capture("echo " + str(i) + " >> " + self.temp_file).has_next()
+
+    def check_iter_wrapper(self):
+        output = self.account.ssh_capture("tail " + self.temp_file)
+        for i in range(self.line_num):
+            assert output.has_next() == True
+            assert output.next().strip() == str(i)
+        start = time.time()
+        assert output.has_next() == False
+        stop = time.time()
+        assert stop - start < 1, "has_next() should return immediately"
+
+    def check_iter_wrapper_timeout(self):
+        output = self.account.ssh_capture("tail -F " + self.temp_file)
+        for i in range(self.line_num):
+            assert output.has_next() == True
+            assert output.next().strip() == str(i)
+        start = time.time()
+        assert output.has_next(timeout_sec=5) == False
+        stop = time.time()
+        assert (stop - start >= 5) and (stop - start) < 6, "has_next() should return right after 5 seconds"
```

Comment:
Similar to above - you could probably safely tighten the bound of this check (closer to 5)